### PR TITLE
Stake banner

### PR DESCRIPTION
**Pull request by ZaHuPro**
This PR requires a new environment variable **STAKE_URL**

### CODE REVIEW

File: infra/token-transfer-client/src/constants/index.js
Comment (franckc) [inline infra/token-transfer-client/src/constants/index.js:16]: This must be your IP? Let's keep localhost since most developers will be running the stack on their local environment. But we could have an override to make it easier to run the API remotely. Something like:
```
apiUrl = process.env.DEV_API_URL || 'http://localhost:5000'
```

Comment (ZaHuPro) [reply infra/token-transfer-client/src/constants/index.js:16]: @franckc  Sorry, My bad. I will update this

Comment (ZaHuPro) [reply infra/token-transfer-client/src/constants/index.js:16]: @franckc I have changed this

Comment (franckc) [reply infra/token-transfer-client/src/constants/index.js]: no problem! thanks for fixing it.

**Review by franckc — APPROVED** [review]
LGTM

**Comment by franckc** [comment]
@micahalcorn @tomlinton Could you please do a review on these T3 changes before we merge them in?

**Comment by micahalcorn** [comment]
I'll defer to @tomlinton since this is his baby and he knows how to reproduce all of the relevant states. I read through the changes and they seem to address the known product requirements. 👍

**Comment by tomlinton** [comment]
@ZaHuPro are the changes to the package versions and yarn.lock required? If possible we should revert those as I'd prefer not to change any dependencies.

File: infra/token-transfer-client/src/components/pages/Dashboard.js
Comment (tomlinton) [inline infra/token-transfer-client/src/components/pages/Dashboard.js:30]: This or the above condition will need to be changed otherwise the CTA will never show. I think one or the other should just be set to `true` depending on if we want the full width CTA or the square box one.

Comment (micahalcorn) [reply infra/token-transfer-client/src/components/pages/Dashboard.js:30]: My understanding was that the square one would be displayed for people who had no existing lock-ups and the full-width banner would display if the square box was displaying existing lock-up information.

Comment (tomlinton) [reply infra/token-transfer-client/src/components/pages/Dashboard.js:30]: Ok then I think just the `data.config.earlyLockupsEnabled` needs to be removed from that conditional.

**Comment by ZaHuPro** [comment]
@tomlinton  The older versions of SASS are not compatible with the latest macOS version

**Review by tomlinton — APPROVED** [review]

**Pull request merged**
